### PR TITLE
[CVE-2020-7753] Update remark to 13.0.0

### LINKED ISSUE #1706: [CVE-2020-7753] Update remark to 13.0.0
I was tasked with resolving [a vulnerability coming from `trim` package](https://snyk.io/vuln/SNYK-JS-TRIM-1017038). Dependency chain is as follows: `trim` <- `remark-parse` <- `remark` <- `react-styleguidist`<- our repo.

`remark-parse` in version 9.0.0 [removes the `trim` completely](https://github.com/remarkjs/remark/commit/6b42465526bb15f70d98a0ea0daccea01ffb8004#diff-0828111e0233dec7891d8f0d6a9f496957215446a0935a1c71e22cc6c6c1760c), and to have `remark-parse` in 9.0.0, we need to bump the `remark` to 13.0.0.

Unfortunately, `react-styleguidist` fixes the `remark` to ^11.0.1.

Because of the fact that `trim`'s vulnerable code won't be used anyway in modern browsers, I've managed to reduce the vulnerability from **high** to **low**, but it would be better if we could get rid of `trim` altogether - hence I'm opening an issue on your side.

### PR DESCRIPTION
**Pull request by poteirard**
Fixes #1706

### CODE REVIEW

**Comment by sapegin** [comment]
Thanks for the pull request! Please have a look at the Travis CI results.

**Comment by poteirard** [comment]
Fixed the emphasis tags. I asked in the remark repo how to have the auto links https://github.com/remarkjs/remark/discussions/560

File: src/loaders/utils/chunkify.ts [hunk 1]
```diff
@@ -64,13 +64,14 @@ export default function chunkify(
 	}
 
 	const rendered = remark()
+		.use({ settings: { emphasis: '_' } })
```
Comment (sapegin) [inline src/loaders/utils/chunkify.ts:67]: Let's keep the default value, it doesn't really matter for us.

**Review by sapegin — COMMENTED** [review]
The autolinking I think is actually undesirable and the new behavior is better. I found that even remove this behavior manually:

https://github.com/styleguidist/react-styleguidist/blob/master/src/loaders/utils/noAutoLinkRemarkPlugin.ts

So I think we probably can remove this file now.

File: src/loaders/utils/chunkify.ts (hunk 1 above)
Comment (poteirard) [reply src/loaders/utils/chunkify.ts]: Removed and snapshots updated

**Comment by sapegin** [comment]
Could you try to remove `noAutoLinkRemarkPlugin.ts`?

**Comment by poteirard** [comment]
Removed noAutoLinkRemarkPlugin

**Comment by sapegin** [comment]
Thanks!

**Pull request merged**